app/controllers/benchmark_runs_controller.rb

```diff
@@ -31,6 +31,12 @@ def create
     benchmark_run.result = params[:benchmark_run][:result]
     benchmark_run.save!
 
+    if params[:commit_hash]
+      BenchmarkRegressionJob.new.perform(commit_hash: params[:commit_hash], category: benchmark_type_params[:category],
+      script_url: benchmark_type_params[:script_url], name: benchmark_result_type_params[:name],
+      unit: benchmark_result_type_params[:unit], repos: params[:repo], organizations: params[:organization] )
+    end
```

Comment:
Can this be more optimized? Also, for writing the tests I need to change the benchmark results and check. But with this parameters how can I write tests?